docs/Fields.md

```diff
@@ -669,54 +677,106 @@ import { UrlField } from 'react-admin';
 
 ## Styling Fields
 
-All field components accept the `style` prop, which overrides the default style of the field *container*:
+All field components accept a `className` prop, allowing you to customize their style to your liking. We advise you to use Material UI styling solution, JSS to generate those classes. See their [documentation](https://material-ui-next.com/customization/css-in-js/#api) about that.
```

Comment:
s/to use Material UI styling solution, JSS to generate those classes/to use the Material UI styling solution, JSS, to generate those classes/